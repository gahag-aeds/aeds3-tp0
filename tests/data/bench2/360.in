360 180
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 
Sub 202 321
Max 194 313
Add 213 332
Min 91 210
Sub 112 231
Sum 213 332
Sub 158 277
Max 20 139
Add 211 330
Sum 15 134
Add 229 348
Min 208 327
Add 31 150
Sum 171 290
Sub 28 147
Max 2 121
Sub 26 145
Sum 105 224
Sub 172 291
Max 113 232
Add 85 204
Max 14 133
Sub 156 275
Min 175 294
Sub 66 185
Min 229 348
Sub 38 157
Max 215 334
Add 201 320
Min 130 249
Add 139 258
Min 34 153
Sub 196 315
Sum 111 230
Add 237 356
Max 179 298
Add 195 314
Min 154 273
Add 199 318
Min 37 156
Add 41 160
Max 59 178
Sub 4 123
Sum 183 302
Add 229 348
Max 143 262
Sub 184 303
Sum 126 245
Sub 120 239
Max 227 346
Sub 110 229
Min 76 195
Sub 112 231
Max 26 145
Sub 122 241
Sum 177 296
Sub 126 245
Min 160 279
Add 23 142
Sum 198 317
Add 49 168
Max 161 280
Sub 104 223
Max 116 235
Add 143 262
Min 100 219
Add 55 174
Max 209 328
Add 125 244
Max 125 244
Add 117 236
Min 37 156
Add 183 302
Max 233 352
Add 219 338
Min 43 162
Add 135 254
Min 34 153
Sub 40 159
Sum 126 245
Sub 132 251
Sum 150 269
Add 201 320
Sum 3 122
Add 47 166
Max 83 202
Add 51 170
Min 172 291
Sub 114 233
Min 73 192
Add 1 120
Sum 162 281
Add 105 224
Max 104 223
Sub 150 269
Max 119 238
Add 203 322
Sum 204 323
Add 199 318
Max 200 319
Sub 200 319
Min 76 195
Sub 236 355
Max 14 133
Sub 68 187
Sum 87 206
Sub 56 175
Max 74 193
Sub 120 239
Max 95 214
Sub 72 191
Sum 12 131
Sub 116 235
Max 32 151
Sub 126 245
Sum 162 281
Sub 226 345
Sum 177 296
Add 93 212
Sum 99 218
Add 121 240
Min 94 213
Add 133 252
Max 98 217
Add 69 188
Sum 42 161
Sub 88 207
Max 32 151
Add 117 236
Max 47 166
Add 103 222
Min 76 195
Sub 122 241
Sum 210 329
Add 89 208
Min 61 180
Sub 56 175
Min 16 135
Sub 6 125
Max 176 295
Sub 222 341
Max 77 196
Add 59 178
Max 98 217
Add 109 228
Min 184 303
Add 19 138
Max 206 325
Sub 232 351
Max 224 343
Add 177 296
Sum 225 344
Add 189 308
Sum 69 188
Sub 194 313
Max 17 136
Sub 222 341
Max 41 160
Sub 160 279
Max 98 217
Add 87 206
Max 134 253
Add 173 292
Max 80 199
Sub 104 223
Min 133 252
Sub 140 259
Sum 159 278
Sub 148 267
Max 146 265
Sub 206 325
Sum 129 248
Sub 222 341
Sum 24 143
Sub 98 217
Min 202 321
Sub 80 199
Min 229 348
Sub 40 159
Sum 183 302
