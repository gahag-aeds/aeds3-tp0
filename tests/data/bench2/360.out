97
0
6043
99
5750
1
6168
99
5721
97
99
-3
-1
102
-4
-2
5766
99
-3
-2
97
6397
102
5991
104
-5
96
5979
-8
6171
101
92
-8
104
98
1
104
-1
1
5942
6269
5726
96
-6
-7
6437
97
96
6716
106
-7
101
5697
99
96
5696
98
6120
6012
5230
-10
94
5559
99
99
-7
6498
-7
-1
104
97
95
2
103
102
6335
5608
99
99
95
101
98
-5
6202
100
5731
5794
-2
-2
5951
